README.md

```diff
@@ -228,7 +227,7 @@ signed-manifest = {
 
 manifest = {
   metadata: manifest-metadata,
-  resources: [* [resource: resource-key, hashes: hashes]],
+  resource-hashes: {* hash-algorithm => [hash-value]},
```

Comment:
Is the mapping resource->hash done here by using the same order of the hash-values as resources in indexed content? It's not explicitly said anywhere I think...